pkg/scheduler/objects/node.go

```diff
@@ -239,6 +239,27 @@ func (sn *Node) GetAvailableResource() *resources.Resource {
 	return sn.availableResource.Clone()
 }
 
+// Get the utilized resource on this node.
+func (sn *Node) GetUtilizedResource() *resources.Resource {
+	total := sn.GetCapacity()
+	resourceAllocated := sn.GetAllocatedResource()
+	m := make(map[string]bool)
+	utilizedResource := make(map[string]resources.Quantity)
+
+	for name, v := range total.Resources {
+		if v > 0 {
+			m[name] = true
+		}
+	}
```

Comment:
Can we avoid this entire for loop and check each res value of "total" directly in below for loop?